### PR TITLE
Rename to "Deployment Best Practices".

### PR DESCRIPTION
**Pull request by patrickvacek**
I have not changed any of the URLs nor the published 1.0.0 version of the HTML stored in this repository.

Arguably depends on https://github.com/uptane/deployment-considerations/pull/58.

### CODE REVIEW

**Review by mnm678 — DISMISSED** [review]

File: uptane-standard.md
Comment (jhdalek55) [inline uptane-standard.md:490]:
```suggestion
In addition to the required metadata, Targets metadata files MAY contain extra metadata for images on the repository. This metadata can be customized for a particular use case. Examples of use cases for different types of custom metadata can be found in the  *Deployment Best Practices* document ({{DEPLOY}}). However, there are a few important pieces of custom metadata that SHOULD be present in most implementations. In addition, there is one element in the custom metadata that MUST be present in the Targets metadata from the Director.
```

File: uptane-standard.md
Comment (jhdalek55) [inline uptane-standard.md:652]:
```suggestion
All ECUs MUST verify image metadata as specified in {{metadata_verification}} before installing an image or making it available to other ECUs. A Primary ECU MUST perform full verification ({{full_verification}}). A Secondary ECU SHOULD perform full verification if possible. If a Secondary cannot perform full verification, it SHALL, at the very least, perform partial verification. In addition, it MAY also perform some steps from the full verification process. See the *Uptane Deployment Best Practices* document ({{DEPLOY}}) for a discussion of how to choose between partial and full verification.
```

File: uptane-standard.md
Comment (jhdalek55) [inline uptane-standard.md:654]:
```suggestion
ECUs MUST have a secure source of time. An OEM/Uptane implementor MAY use any external source of time that is demonstrably secure. The *Uptane Deployment Best Practices* document ({{DEPLOY}}) describes one way to implement an external time server to cryptographically attest time, as well as the security properties required. When "loading time" is referenced in procedures in this Standard, it should be understood to mean loading into memory the current time (if the ECU has its own secure clock), or the most recent attested time.
```

File: uptane-standard.md
Comment (jhdalek55) [inline uptane-standard.md:660]:
```suggestion
1. A sufficiently recent copy of required Uptane metadata at the time of manufacture or install. This is necessary for the ECU to authenticate that the remote repository is legitimate when it first downloads metadata in the field. See the *Uptane Deployment Best Practices* document ({{DEPLOY}}) for more information.
```

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
Left a few small edits. I think we need to consistently follow the "Best Practices" title with the word "document" as we are now treating this a stand-alone document.

**Comment by patrickvacek** [comment]
> Left a few small edits. I think we need to consistently follow the "Best Practices" title with the word "document" as we are now treating this a stand-alone document.

Good point. I was a bit unsure about removing the word anyway. Thanks for the typo fix, too! I've merged all your suggestions and rebased them into one commit.

**Comment by jhdalek55** [comment]
I think you can merge this @patrickvacek unless you want one more review not from NYU or HERE.

**Comment by patrickvacek** [comment]
It still requires an actual approval before it'll let me merge. :)

**Review by jhdalek55 — APPROVED** [review]
Ok. Here's my approval

**Comment by patrickvacek** [comment]
Oops, I guess I don't have the rights after all. Strange. I can open PRs directly in this repo, but I guess I can't merge.

**Comment by jhdalek55** [comment]
Hmmm...I thought I did this in the past. Well, we know @JustinCappos and @tkfu can merge so, could one of you take care of this merge, if you have no objections.

**Review by tkfu — APPROVED** [review]

**Comment by tkfu** [comment]
> Oops, I guess I don't have the rights after all. Strange. I can open PRs directly in this repo, but I guess I can't merge.

I think this is related to the role changes github pushed out last year. I've now changed everyone who formerly had write permissions to the "maintain" role; I think that should solve it.

**Comment by jhdalek55** [comment]
Thanks for the quick assist, @tkfu .

**Pull request merged**
